scripts/udp_bidirectional_flow.py

```diff
@@ -0,0 +1,138 @@
+import snappi
+from snappi_test.tests import utils
+
+
+def udp_bidirectional_flow():
+    """
+    This script does following:
+    - Send 1000 packets back and forth between the two ports at a rate of
+      1000 packets per second.
+    - Validate that total packets sent and received on both interfaces is as
+      expected using port metrics.
+    - Validate that captured UDP packets on both the ports are as expected.
+    """
+    # create a new API instance where location points to controller
+    # location = "https://<tgenip>:<443(or)11009(or)port_set_by_user>"
+    # ext = ixnetwork (or) none for Athena
+    api = snappi.api(location="https://localhost:443", verify=False, ext=None)
+    # and an empty traffic configuration to be pushed to controller later on
+    cfg = api.config()
+
+    # add two ports where location points to traffic-engine (aka ports)
+    # port.location = [athena] localhost:5555 or [ixnetwork] \
```

Comment:
simplify here too. please don't use any notations like [ixnetwork], etc.

Reply:
# add two ports where location points to traffic-engine (aka ports)
    # ixia-c, trex:
    #   port.location = "localhost:5555"
    # ixnetwork:
    #   port.location = "<chassisip>;card;port"
    #

Reply:
is it fine with the above way?